# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/vr/ipc/VRProcessParent.cpp
# Commit: 75b04de7e99c
# Full Hash: 75b04de7e99cab7bed5cc8b0f2f6d52ddf174bf3
# Author: Jean-Yves Avenard <jyavenard@mozilla.com>
# Date: 2019-05-25 09:39:13
# Regressor Bug: 1550422
# File Overlap Count: 1
# Description:
#   Bug 1550422 - P15. Move Skip and Once gfxPrefs to StaticPrefs. r=jrmuizel
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D31259
# ==============================================================================

diff -r 91c3acdb2454 -r 75b04de7e99c gfx/vr/ipc/VRProcessParent.cpp
--- a/gfx/vr/ipc/VRProcessParent.cpp	Fri May 24 11:32:20 2019 +0000
+++ b/gfx/vr/ipc/VRProcessParent.cpp	Fri May 24 11:32:54 2019 +0000
@@ -11,6 +11,7 @@
 #include "mozilla/gfx/GPUChild.h"
 #include "mozilla/ipc/ProtocolTypes.h"
 #include "mozilla/ipc/ProtocolUtils.h"  // for IToplevelProtocol
+#include "mozilla/StaticPrefs.h"
 #include "mozilla/TimeStamp.h"          // for TimeStamp
 #include "mozilla/Unused.h"
 #include "ProcessUtils.h"
@@ -81,7 +82,7 @@
     return !!mVRChild;
   }
 
-  int32_t timeoutMs = gfxPrefs::VRProcessTimeoutMs();
+  int32_t timeoutMs = StaticPrefs::VRProcessTimeoutMs();
 
   // If one of the following environment variables are set we can effectively
   // ignore the timeout - as we can guarantee the compositor process will be